apps/registry/pages/api/formatters/template.js

```diff
@@ -3,70 +3,36 @@ export const THEMES = {
-  direct: require('jsonresume-theme-direct'),
-  dinesh: require('jsonresume-theme-dinesh'),
   elegant: require('jsonresume-theme-elegant'),
-  elite: require('jsonresume-theme-elite'),
-  // eloquent: require('jsonresume-theme-eloquent'),
   'el-santo': require('jsonresume-theme-el-santo'),
   even: require('jsonresume-theme-even'),
-  flat: require('jsonresume-theme-flat'),
-  'flat-fr': require('jsonresume-theme-flat-fr'),
-  fresh: require('jsonresume-theme-fresh'),
-  full: require('jsonresume-theme-full'),
-  joeytall: require('jsonresume-theme-joeytall'),
   github: require('jsonresume-theme-github'),
   github2: require('jsonresume-theme-github2'),
   jacrys: require('jsonresume-theme-jacrys'),
   kards: require('jsonresume-theme-kards'),
-  keloran: require('jsonresume-theme-keloran'),
   kendall: require('jsonresume-theme-kendall'),
   macchiato: require('jsonresume-theme-macchiato'),
   mantra: require('jsonresume-theme-mantra'),
   'mocha-responsive': require('jsonresume-theme-mocha-responsive'),
   minyma: require('jsonresume-theme-minyma'),
-  modern: require('jsonresume-theme-modern'),
   msresume: require('jsonresume-theme-msresume'),
   one: require('jsonresume-theme-one'),
-  onepage: require('jsonresume-theme-onepage'),
-  'onepage-plus': require('jsonresume-theme-onepage-plus'),
-  'onepage-efficient': require('jsonresume-theme-onepage-efficient'),
   onepageresume: require('jsonresume-theme-onepageresume'),
   orbit: require('jsonresume-theme-orbit'),
   paper: require('jsonresume-theme-paper'),
   'paper-plus-plus': require('jsonresume-theme-paper-plus-plus'),
-  papirus: require('jsonresume-theme-papirus'),
   pumpkin: require('jsonresume-theme-pumpkin'),
   rocketspacer: require('jsonresume-theme-rocketspacer'),
-  short: require('jsonresume-theme-short'),
   'simple-red': require('jsonresume-theme-simple-red'),
-  relaxed: require('jsonresume-theme-relaxed'),
-  resu: require('jsonresume-theme-resu'),
   rickosborne: require('jsonresume-theme-rickosborne'),
-  riga: require('jsonresume-theme-riga'),
-  slick: require('jsonresume-theme-slick'),
-  sceptile: require('jsonresume-theme-sceptile'),
-  simple: require('jsonresume-theme-simple'),
-  spartacus: require('jsonresume-theme-spartacus'),
-  'timeline-fixed': require('jsonresume-theme-timeline-fixed'),
   spartan: require('jsonresume-theme-spartan'),
-  srt: require('jsonresume-theme-srt'),
   stackoverflowed: require('jsonresume-theme-stackoverflowed'),
   stackoverflow: require('jsonresume-theme-stackoverflow'),
   standard: require('jsonresume-theme-standard'),
   'standard-resume': require('jsonresume-theme-standard-resume'),
-  'tachyons-clean': require('jsonresume-theme-tachyons-clean'),
   'tan-responsive': require('jsonresume-theme-tan-responsive'),
   techlead: require('jsonresume-theme-techlead'),
-  verbum: require('jsonresume-theme-verbum'),
-  wraypro: require('jsonresume-theme-wraypro'),
 };
 
 const getTheme = (theme) => {
```

Comment:
That is a nice observation. However totally out of scope of the PR. Just open a pull request that fixes this, if you are concerned about this. Coming in and reviewing existing code, that you are not tasked to use or change, like this is considered quite rude in most projects.

Reply:
@levino you know that's a bot yeah? aha

Reply:
Thank you for the heads up, @thomasdavis. I know that @coderabbitai is a bot. Supposedly an "intelligent" bot, as far as I can tell. I am trying things out with it. And see: It claims it would adapt and learn:
>@levino Understood, I apologize for the overreach. I'll refrain from reviewing unrelated code changes in the future. Thank you for the guidance.

I find the style a bit too obedient (like with ChatGPT where it always apologizes for everything) but it says it has understood. Lets see if it remembers too!

BTW: If I had not understood that it is a bot, I would probably had asked it to approve the PR 😉

Reply:
@levino aha sorry. and yeah, cool idea for an experiment. 

@coderabbitai are you able to create a PR for the feedback that you originally left?